### PR TITLE
Docker configuration update - No more issues with file permissions 

### PR DESCRIPTION
**Pull request by funder7**
## Description

I started to work with this starter project, but using docker, every time that I launch a command (eg. generate a migration / model), then the generated files result created by `root` on my host machine, and are not editable in turn. 

To avoid this problem, I followed [this guide](https://semaphoreci.com/community/tutorials/dockerizing-a-ruby-on-rails-application).
After rebuilding the images, and launching docker-compose, now each service uses "user" instead of "root". "user" shares the same UID/GID as my host user (1000/1000), and so generated files are finally editable on the host machine, without any change of permissions.

I've included some Readme info, but English is not my main language, so please, feel free to update the description if this PR will be approved. 
 
Hope that you will find this useful, as I do :-) 

Cheers
Federico

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,4 +1,3 @@
 # Docker user and group ids
-# On linux 1000 is the id of the first
```
Comment (funder7) [inline .env.sample]: typo

**Comment by damianlegawiec** [comment]
Hi @funder7 

There is some problem with permission when running `bundle install` inside the container - 

```bash
user@330b0ae308a5:/app$ bundle install
Using rake 13.0.3
Errno::EACCES: Permission denied @ rb_sysopen - /bundle/bin/rake
```

**Comment by funder7** [comment]
Hello @damianlegawiec, 
You're right, I noticed that yesterday, now I've pushed an update, now it's possible to run `bundle install` also from inside of the container without issues.

Also I had a problem with the `.env` file: starting the project without it, lead to docker-compose creating it, but as a folder. That blocked the correct startup of the `web` service. Probably it's a secondary issue as the setup script creates the `.env` file.
Anyway I removed it from the `docker-compose.yml` file, as it's already included in the `/app` folder from what I can see.

Basically now there are no more issues related to this, as, if the file exists, then it's copied like the other files in the project. If it doesn't exist, then nothing happens and every docker service can start regularly. 

I hope that now everything is fixed!

File: Dockerfile.development [hunk 2]
```diff
@@ -3,9 +3,10 @@ FROM ruby:2.7.3
 # SETUP USER
 ARG USER_ID
 ARG GROUP_ID
+ARG USER_NAME=user
```
Comment (funder7) [inline Dockerfile.development:6]: the regular user name can be overridden

File: docker-compose.yml [hunk 3]
```diff
@@ -26,7 +26,6 @@ services:
     volumes:
       - 'bundle_cache:/bundle'
       - '.:/app'
```
Comment (funder7) [inline docker-compose.yml:28]: this already includes the `.env` file (if present)
